lib/saito/peer.js

```diff
@@ -404,24 +408,29 @@ console.log("old keyist request");
         }
     }
 
-    async sendResponseFromStr(message_id, data) {
-        let data_to_send = {
-            message_name: "",
-            message_id: message_id,
-            message_data: data
-        };
-        let buffer = Buffer.from(JSON.stringify(data_to_send), "utf-8");
+    // async sendResponseFromStr(message_id, data) {
+    //     console.log("sendResponseFromStr");
+    //     console.log(data);
+    //     if (this.socket && this.socket.readyState === this.socket.OPEN) {
+    //         await this.app.networkApi.sendAPIResponse(this.socket, "RESULT__", message_id, Buffer.from(data, 'utf-8'));
+    //     } else {
+    //         console.error("socket not open");
+    //         console.log(this);
+    //     }
+    // }
+    async sendResponse(message_id, data) {
+        console.log("sendResponse");
```

Comment:
Could save a log by doing `console.log("sendResponse", data);`

Reply:
thanks.  actually this log will be probably removed.